system/ee/EllisLab/ExpressionEngine/Service/Validation/Rule/ExactLength.php

```diff
@@ -28,12 +31,8 @@ public function validate($key, $value)
 			return FALSE;
 		}
 
-		if (function_exists('mb_strlen'))
-		{
-			return (mb_strlen($value) != $length) ? FALSE : TRUE;
-		}
```

Comment:
This switched from `!=` to `<` so is the result correct if the `$value` is longer than length? Also, it's a confusing conditional to check for truthiness to return false. I know it was like this originally but shouldn't this just be `return (ee_mb_strlen($value) == $length);` or `return (ee_mb_strlen($value) > $length);` ?

Reply:
Good catch, fixed on latest commit.